src/Microsoft.AspNet.Mvc/RazorPreCompileModule.cs

```diff
@@ -84,6 +76,27 @@ public void AfterCompile(IAfterCompileContext context)
         {
         }
 
+        // REVIEW: Welcome back to life delegating SP!!!
```

Comment:
ick!  we just got rid of this hack.  why doesn't Hosting support the approaches MVC requires?